no-computer.md

```diff
@@ -1,55 +1,68 @@
 # No computer project
-
+Baseball! It's TOTALLY a DSL.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This language would be good for people who want to run baseball simulations to ask "what if" questions. If a pitcher throws the ball a certain way to a certain location to a certain hitter who swings a certain way, what will happen?
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea would help baseball executives and casual fans alike entertain possibilities of things that would happen in a game. For an employee of a baseball team, it would help them prepare for games and opposing pitchers/hitters by "seeing what would happen" ahead of time. It could also help settle petty arguments between fans or help them see what would happen if their favorite player did something specific.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because so much language and terminology is specific to baseball. We can define these things specifically in a DSL, which makes it seem like the right approach. We can define the field, the players, the strike zone, and so much more as well.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I LOVE baseball and this is totally something I would use! I think baseball, which is one of the stats-heaviest sports I can think of, lends itself well to computation and simulation well.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Helping people simulate baseball events
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I think the user would input either existing players (whose hitting zones, pitch preferences etc would come directly from the internet) or create players within the program, as well as a pitcher with certain pitches, pitch speed, and so on, and then ask at what likelihood different events would occur (single, strikeout etc) if a pitcher through a certain pitch aimed for a certain area. This is the right way to interact with the domain because the user is given the ability to specify exactly the desired event.
+
+The DSL is massively extensible, which is both a blessing and a curse. The field itself can be broken up into (arbitrarily small) zones and then, if a physics engine is applied, we could actually simulate where a ball would go, with what exit velocity, and so on.
```

Comment:
Although I have never played any of those fantasy sports games, how might this be different from Fantasy Baseball?